doc/classes/AudioEffectSpectrumAnalyzer.xml

```diff
@@ -1,31 +1,43 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectSpectrumAnalyzer" inherits="AudioEffect" version="4.0">
 	<brief_description>
+		Audio effect that can be used for near-real-time audio visualizations.
 	</brief_description>
 	<description>
+		This audio effect does not affect sound output, but can be used for near-real-time audio visualizations.
+		See also [AudioStreamGenerator] for procedurally generating sounds.
 	</description>
 	<tutorials>
+		<link title="https://godotengine.org/asset-library/asset/528">Audio Spectrum Demo</link>
+		<link title="https://godotengine.org/article/godot-32-will-get-new-audio-features">Godot 3.2 will get new audio features</link>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="buffer_length" type="float" setter="set_buffer_length" getter="get_buffer_length" default="2.0">
+			The length of the buffer to keep (in seconds). Higher values keep data around for longer, but require more memory.
 		</member>
 		<member name="fft_size" type="int" setter="set_fft_size" getter="get_fft_size" enum="AudioEffectSpectrumAnalyzer.FFT_Size" default="2">
+			The size of the [url=https://en.wikipedia.org/wiki/Fast_Fourier_transform]Fast Fourier transform[/url] buffer. Higher values result in better quality, but are more demanding on the CPU and may cause audio cracking if the CPU can't keep up.
 		</member>
 		<member name="tap_back_pos" type="float" setter="set_tap_back_pos" getter="get_tap_back_pos" default="0.01">
 		</member>
```

Comment:
Does anyone know what this property is used for? I'd like to get 100% completion :slightly_smiling_face: